fem/linearform.hpp

```diff
@@ -82,7 +86,10 @@ class LinearForm : public Vector
    /// Creates linear form associated with FE space @a *f.
    /** The pointer @a f is not owned by the newly constructed object. */
    LinearForm(FiniteElementSpace *f) : Vector(f->GetVSize())
-   { fes = f; ext = nullptr; extern_lfs = 0; UseDevice(true); }
+   {
+      fes = f; ext = nullptr; extern_lfs = 0; UseDevice(true);
```

Comment:
I suggest moving each statement here to its own line (and perhaps moving the function definition to the cpp file).

Reply:
Addressed in https://github.com/mfem/mfem/pull/3235/commits/c60263c1fd9ecaa9a437346bbb69e172939d7fa1